### PR TITLE
Migrate to new documentation layout

### PR DESCRIPTION
**Pull request by petesfrench**
Migrate to use https://vanillaframework.io/docs/layouts/documentation

### CODE REVIEW

**Comment by petesfrench** [comment]
@bartaz i think you are most familiar with these types of migrations. Would you be able to take a look please?

File: static/sass/main.scss [hunk 1]
```diff
@@ -16,3 +16,7 @@
 @include multipass-github-buttons;
 @include multipass-p-strip;
 @include multipass-p-tabs;
+
+.p-navigation__banner {
+  padding-left: 0;
```
Comment (bartaz) [inline static/sass/main.scss:21]: I don't think it's needed anymore with proper use of `p-navigation__row--25-75`, as we already set it in Vanilla:

image

Comment (bartaz) [reply static/sass/main.scss:21]: And it actually causes a bug on docs layout. This should be removed, so the padding 0 is only applied when needed (by Vanilla):

image

File: templates/_layouts/_footer.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
+{% if is_docs %}
+<footer class="p-footer p-strip">
```
Comment (bartaz) [inline templates/_layouts/_footer.html:2]: I think it works better when footer is dark to diffrenciate from the content/sidebar above.

```suggestion
<footer class="p-footer p-strip is-dark">
```

**Comment by petesfrench** [comment]
thanks @bartaz, updated

**Pull request merged**